cmd/zpool/zpool_main.c

```diff
@@ -3393,8 +3415,17 @@ print_vdev_stats(zpool_handle_t *zhp, const char *name, nvlist_t *oldnv,
 		print_iostat_histos(cb, oldnv, newnv, scale, name);
 	}
 
-	if (!(cb->cb_flags & IOS_ANYHISTO_M))
+
+	if (!(cb->cb_flags & IOS_ANYHISTO_M)) {
+		if (cb->vcdl) {
+			char *path;
+			if (nvlist_lookup_string(newnv, ZPOOL_CONFIG_PATH,
+			    &path) == 0)
+				zpool_print_cmd(cb->vcdl, zpool_get_name(zhp),
+				    path);
+		}
```

Comment:
It's there so we don't print the output of the command with histograms (which looks weird), and makes the printing logic slightly easier.  I'll add a note in the man page about this.